frame/bounties/src/benchmarking.rs

```diff
@@ -91,100 +91,117 @@ fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
 	assert_eq!(event, &system_event);
 }
 
-fn setup_subbounty<T: Config>(u: u32, d: u32) -> (
-	T::AccountId,
-	T::AccountId,
-	T::AccountId,
-	BalanceOf<T>,
-	BalanceOf<T>,
-	BalanceOf<T>,
-	BalanceOf<T>,
-	Vec<u8>,
-) {
+#[derive(Clone)]
+struct BmSubbountyCfg<T: Config> {
```

Comment:
make it straightforward, `BenchmarkSubBountyConfig` or `SubBountyConfig`

Reply:
Thanks for suggestion Bryan, switched to "BenchmarkSubBountyConfig"